### PR TITLE
Ltc

### PR DESCRIPTION
**Pull request by enriquesouza**
LTC, we are fetching the price, showing on a list, sending, calculating the Mac, from USD, and to USD, reviewing, listing the transactions, and broadcasting.

### CODE REVIEW

File: voltixApp/VoltixApp/Model/Litecoin/Litecoin.swift
Comment (Rockindash) [inline voltixApp/VoltixApp/Model/Litecoin/Litecoin.swift:1]: Can rename the file to LitecoinExtension to make it more clear.

File: voltixApp/VoltixApp/Model/UTXOTransactionMempool.swift
Comment (Rockindash) [inline voltixApp/VoltixApp/Model/UTXOTransactionMempool.swift:1]: Suggestion. Can add a static variable called example with predefined parameters. It is quite useful as test data for previews. Can check Vault model for reference.

File: voltixApp/VoltixApp/Model/Litecoin/Litecoin.swift
Comment (enriquesouza) [reply voltixApp/VoltixApp/Model/Litecoin/Litecoin.swift:1]: Done

File: voltixApp/VoltixApp/View Models/CoinViewModel.swift [hunk 1]
```diff
@@ -1,60 +1,74 @@
...
+	@Published var coinBalance = "0.0"
+	
+	func loadData(utxoBtc: UnspentOutputsService, utxoLtc: LitecoinUnspentOutputsService, eth: EthplorerAPIService, thor: ThorchainService, tx: SendTransaction) async {
+		print("realoading data...")
+		isLoading = true
+		
+		if tx.coin.chain.name.lowercased() == Chain.Bitcoin.name.lowercased() {
+			await utxoBtc.fetchUnspentOutputs(for: tx.fromAddress)
+		}
+		else if tx.coin.chain.name.lowercased() == Chain.Litecoin.name.lowercased() {
```
[the remark this answers is not in the document]
Comment (enriquesouza) [reply voltixApp/VoltixApp/View Models/CoinViewModel.swift]: Sure, done

**Pull request merged**